app/options/apigw.cue

```diff
@@ -25,6 +25,14 @@ apigw: schema.#optionsGroup & {
 			type:        "bool"
 			description: "Enable extra logging"
 		}
+		profiler_enabled: {
+			type:        "bool"
+			description: "Enable profiler"
+		}
+		profiler_global: {
+			type:        "bool"
+			description: "Profiler enabled for all routes"
```

Comment:
do we need to explain in more detail gow enabling this affects profilers on routes?

Reply:
and, should we log this into stdout (warn/info/debug) that global profiler is enabled?

Reply:
> do we need to explain in more detail gow enabling this affects profilers on routes?
I'll add a little more info on server logs, most will be in docs

> and, should we log this into stdout (warn/info/debug) that global profiler is enabled?
yes